### PR TITLE
Issue #558 Keepachangelog labels

### PR DESCRIPTION
**Pull request by jennyknuth**
This PR addresses issue #558 and adds support for the six standard categories of changes listed in [keepachangelog](https://keepachangelog.com/en/1.0.0/) (three more categories—deprecated, removed, security—are added to the three that already exist).

This functionality might be covered by the add_sections option, but I couldn't find any documentation around add_sections or configure_sections so it does not appear to be easy to do. 

Besides, Keepachangelog is the current standard reference for changelogs and arguably the six categories it lists should be included as the standard default sections. Including these standard sections/labels in a changelog should not require a user to implement a custom configuration. 

This is my first contribution to open source, so go easy! But I have found your gem very useful and I found an issue I could address that would help improve changelogs everywhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ You can look for params names from the [parser source code (#setup_parser)](http
 
 ## Features and advantages of this project
 
-- Generate canonical, neat changelog file, followed by [basic changelog guidelines](http://keepachangelog.com) :gem:
+- Generate canonical, neat changelog file, with default sections that follow [basic changelog guidelines](http://keepachangelog.com) :gem:
```
Comment (olleolleolle) [inline README.md]: This wording is clearer, and teaches what this is about way better.

File: lib/github_changelog_generator/parser_file.rb [hunk 2]
```diff
@@ -66,8 +66,10 @@ def extract_pair(line)
       [key.tr("-", "_").to_sym, value.gsub(/[\n\r]+/, "")]
     end
 
-    KNOWN_ARRAY_KEYS = %i[exclude_labels include_labels bug_labels
-                          enhancement_labels breaking_labels issue_line_labels between_tags exclude_tags]
+    KNOWN_ARRAY_KEYS = %i[exclude_labels include_labels
+                          breaking_labels enhancement_labels bug_labels
+                          deprecated_labels removed_labels security_labels
+                          issue_line_labels between_tags exclude_tags]
```
Comment (olleolleolle) [inline lib/github_changelog_generator/parser_file.rb]: Tiny thing: thanks for keeping the lines short - Github diffs are rendered much better when the lines are shorter (~ 80 chars). Kudos!

**Comment by olleolleolle** [comment]
Point of order: Thank you so much for this PR!

(If you are up to documenting the new options, do that in a follow-up PR. This one is completely fine to merge in itself.)

**Comment by olleolleolle** [comment]
Merging!

**Comment by TomasVotruba** [comment]
Hey, thanks for this feature.

I'd like to use but I don't know how. Could you add docs to README/Wiki/Links?

**Comment by olleolleolle** [comment]
Hi there! A short note: the only metadata read by this tool is the Label data. You can add Labels to Issues and to Pull Requests.

Commit messages are not read or used. 

Adding Labels matching these grouped lists of names will present that Issue or PR under a specific heading in the generated Markdown file. 

I hope this helps!

**Comment by TomasVotruba** [comment]
I understand that, but that would kill me for existing projects with years of history :(

 Sort by label is just one (now default) implementation of sorting issues/PRs. How to use own function to sort this by issue or PR message?

I know PHP only

**Comment by hugovk** [comment]
@jennyknuth Thank you for adding this feature, just what I was looking for in this tool. And congratulations on your first contribution to open source! :cake: 🎆

**Pull request merged**